testing-utils/src/lib/utils/dynamic-mocks/config.mock.ts

```diff
@@ -36,6 +36,7 @@ export function minimalConfigMock(
   const outputFile = `${PLUGIN_1_SLUG}.${Date.now()}.json`;
 
   const cfg = coreConfigSchema.parse({
+    persist: persistConfigMock({ outputDir }),
```

Comment:
@BioPhoton This change I was not sure about - I wouldn't meddle with unrelated mocks in this PR. If you insist on this change and checked that it does not break existing tests, feel free to reintroduce this change of course.